game_shared/multiplay_gamerules.cpp

```diff
@@ -810,7 +810,7 @@ bool CMultiplayRules::IsMultiplayer( void )
 				RecentAttackerInfo *pTopAssister = pFFPlayer->GetTopKillAssister( pScorer );
 				if ( pTopAssister )
 				{
-					event->SetInt("killassister", pTopAssister->playerIndex );
+					event->SetInt("killassister", pTopAssister->GetUserID() );

```

Comment:
Does this compile? I'm confused - RecentAttackInfo doesnt have a GetUserId function on?